src/site/HamburgerMenu.test.js

```diff
@@ -9,13 +9,14 @@ document.addEventListener("DOMContentLoaded", function() {
 
   toggleButton.addEventListener("click", function(c) {
     c.preventDefault();
-    closeButton.classList.toggle("menu-active");
     contentWrapper.classList.toggle("menu-active");
     menu.classList.toggle("menu-active");
   });
 
   closeButton.addEventListener("click", function(c) {
     c.preventDefault();
-    menu.classList.toggle("menu-active");
+    contentWrapper.classList.remove("menu-active");
```

Comment:
as mentioned yesterday, lets try and use .active instead of menu-active. Except for maybe the content body where i think  `site-menu-active` may make sense.

Reply:
Updated and pushed up with the most recent commit.